agent/tests/test_agent.c

```diff
@@ -142,7 +142,11 @@ static void test_function_debug_name(TSRMLS_D) {
   func = nr_php_zval_to_function(closure TSRMLS_CC);
   name = nr_php_function_debug_name(func);
 
+#if ZEND_MODULE_API_NO < ZEND_8_4_X_API_NO
   tlib_pass_if_str_equal("closure", "{closure} declared at -:1", name);
+#else
+  tlib_pass_if_str_equal("closure", "{closure:-:1} declared at -:1", name);
+#endif
```

Comment:
What's the motivation behind this change?

Reply:
closures are named differently in PHP 8.4. https://github.com/newrelic/newrelic-php-agent/pull/996/commits/d5cf37baf83ca118cb0c271fa700f08b3d5d48b9